test/metabase/db/schema_migrations_test/impl.clj

```diff
@@ -95,6 +99,10 @@
                 1000.0)))
         (when (re-matches #"^\d+$" id)
           (Integer/parseUnsignedInt id))
+        (when-let [[_ & components ] (re-matches timestamp-migration-id-re id)]
+          (let [[major-version year month date hour minute second] (map #(Integer/parseUnsignedInt %) components)
+                unix-timestamp (-> (t/zoned-date-time year month date hour minute second 0 (t/zone-id "UTC")) .toInstant .getEpochSecond)]
```

Comment:
I think an easier way to parse this would be to just put everything after `v49.` in one capture group and then use `u.date/parse` to parse it to a `LocalDateTime`, then you can use `with-time-zone-same-instant` to put it in UTC

Reply:
awesome, did that in https://github.com/metabase/metabase/pull/36787/commits/11738c0b8f85428a1803681df9ea0876da8c1715